delft3dcontainermanager/tasks.py

```diff
@@ -75,7 +75,7 @@ def get_docker_log(self, container_id, stdout=True, stderr=False, tail=5):
     """
     Retrieve the log of a container and return container id and log
     """
-    client = Client(base_url='http://localhost:4000')
+    client = Client(base_url='unix:///var/run/docker.sock')
```

Comment:
These values are updated by each provision:

- If the provision vars are correct, each provision will update this file correctly regardless of what is committed to this repository
- If I don't commit it, somebody else will in the future, I guess :)